extensions/amp-story/1.0/test/test-amp-story-page.js

```diff
@@ -23,201 +24,171 @@ import {MediaType} from '../media-pool';
 import {Services} from '../../../../src/services';
 import {Signals} from '../../../../src/utils/signals';
 import {
+  addAttributesToElement,
   createElementWithAttributes,
   scopedQuerySelectorAll,
 } from '../../../../src/dom';
 import {installFriendlyIframeEmbed} from '../../../../src/friendly-iframe-embed';
 import {registerServiceBuilder} from '../../../../src/service';
 
-describes.realWin('amp-story-page', {amp: true}, (env) => {
-  let win;
-  let element;
-  let gridLayerEl;
-  let page;
-  let isPerformanceTrackingOn;
+describes.realWin(
+  'amp-story-page',
+  {amp: {extensions: ['amp-story:1.0', 'amp-audio']}},
```

Comment:
Big diff caused by change in indentation caused by prettier when adding `amp-audio` as an extension for this test suite. Only additions were LOC 289 - 320. ('should register amp-audio on layoutCallback` test)

Reply:
```
const extensions = [];
describes.realWin('amp-story-page', {amp: {extensions}}, (env) => {
```

Can you try something like this so we don't get the massive diff?

Reply:
good idea, done!